src/components/com_tjcertificate/media/fields/legend.php

```diff
@@ -10,9 +10,9 @@
 
 // No direct access.
 defined('_JEXEC') or die;
+use Joomla\CMS\Form\FormField;
 use Joomla\CMS\Factory;
 use Joomla\CMS\Language\Text;
-use Joomla\CMS\Form\FormHelper;
```

Comment:
I think you will need this line since the line below is using FormHelper i.e. `FormHelper::loadFieldClass('list');` @divyachaudhari